src/answers-core-re-exports.ts

```diff
@@ -0,0 +1,49 @@
+export {
```

Comment:
I guess api extractor doesn't work with `export *`?

Reply:
API extractor does work with `export *`, but that would add everything exported from answers-core into the bundle for Headless. It seemed a little odd to have things like `provideCore()` exported when it isn't used with Headless, so this file exports only the Core types/interfaces that are actually used in Headless

Reply:
sgtm!